10a-walkthrough-6.Rmd

```diff
@@ -0,0 +1,677 @@
+tibble(
+    student = letters[1:10], 
+    school = rep(letters[11:15], 2), 
+    test_score = sample(0:100, 10, replace = TRUE)
+)
+```
+
+Aggregate data totals up a variable--the varibable `test_score` in this case--to "hide" the student level information. The rows of the resulting dataset represent a group. The group our example is the `school` variable:
+
+```{r aggregate level data example}
+tibble(
+    student = letters[1:10], 
+    school = rep(letters[11:15], 2), 
+    test_score = sample(0:100, 10, replace = TRUE)
+) %>% 
+    # Aggregate by school
+    group_by(school) %>% 
+    summarise(mean_score = mean(test_score))
+```
+
+Notice here that this dataset no longer identifies individual students. 
+
+*Student level data for analysis of local populations. Aggregate data for base rate and context.*
+
+Longitudinal analysis is typically done with student level data because educators are interested in what happens to students over time. So if you cannot access student level data, how do we use aggregate data to offer value to the analytic conversation?
+
+Aggregate data is valuable because it allows us to learn from populations that are larger or different from the local student level population. Think of it as an opportunity to learn from totaled up student data from other states or the whole country.
+
+In his book *Thinking Fast and Slow*, Daniel Kahneman discusses the importance of learning from larger populations, a context he refers to as the base rate. The base rate fallacy is the tendency to focus on conclusions we can draw from immediately available information and to ignore analysis of the larger context. It's the difference between computing how often a student at one school is identified for special education services (student level data) and how often a students are identified for special educations services nationally (base rate data). We can use aggregate data to combat the baserate fallacy by putting what we learn from local student data in the context of surrounding populations. 
+
+For example, consider an analysis of student level data in a school district over time. Student level data allows us to ask questions about our local population: One such question is: Are the rates of special education identification for male students different from other gender identitites *in our district*? This style of question looks *inward* at your own educational system. 
+
+Taking a cue from Daniel Kahneman, we should also ask what this pattern looks like in other states or in the country. Aggregate data allows us to ask questions about a different or larger population: One such question is Are the reates of special education identification for male students different from other gender identities *in the United States*? This style of question looks for answers *outside* your own educational system. The combination of these two lines of inquiry are powerful way to generate new knowledge about the student experience. 
```

Comment:
nice - this is super helpful. After the analysis, I find this description of aggregate and student-level data to be helpful, but still wondering if we could move it to the start of this section - or even @ivelasq's? interested to hear your thoughts.